.github/workflows/tests.yml

```diff
@@ -189,6 +189,29 @@ jobs:
           path: test-apps/e2e/**/test-results/**/trace.zip
           retention-days: 1
 
+  cli:
+    timeout-minutes: 60
+    needs: [changes, build, typescript, unit_front]
+    name: 'cli'
+    runs-on: ubuntu-latest
```

Comment:
Assuming our tests as-written work on windows, it would be great if this could be a matrix of [ubuntu-latest, windows-latest] but we can iterate on that later once we know it's working on ubuntu-latest

Reply:
Good idea, let's wait to have a first working test before doing this